tools/dds/realdds-py/CMakeLists.txt

```diff
@@ -1,10 +1,7 @@
 # License: Apache 2.0. See LICENSE file in root directory.
 # Copyright(c) 2022 Intel Corporation. All Rights Reserved.
 cmake_minimum_required(VERSION 3.1.0)
-
-# Note: we use 'pyrs_dds_server' because Python doesn't deal with '-' in module
-# names very well... see the PROJECT_LABEL below...
-project( pyrs_dds_server )
+project( pyrealdds )
```

Comment:
Naming is not consistent - should be realdds-py

Reply:
Can't use `-` in the name.
The collateral is called `pyrealdds` but I chose to show it in the solution as `realdds-py` because I thought it was more recognizable.
I can change the label in the solution (it doesn't change anything else) if you want them to match.

Reply:
Actually, I think that pyrealdds is better, but you should change in other places to be consistent

Reply:
What other places?

Reply:
If you want the solution, fine -- but let's leave it for the next PR unless I have to run the CI another time...